src/BenchmarkDotNet/Toolchains/DotNetCli/DotNetCliCommand.cs

```diff
@@ -218,7 +218,7 @@ private static string GetMandatoryMsBuildSettings(string buildConfiguration)
         {
             // we use these settings to make sure that MSBuild does the job and simply quits without spawning any long living processes
             // we want to avoid "file in use" and "zombie processes" issues
-            const string NoMsBuildZombieProcesses = "/p:UseSharedCompilation=false /p:BuildInParallel=false /m:1 /p:Deterministic=true";
```

Comment:
Is this flag supported by older MSBuild versions?

Reply:
I'm not sure which version added it, but it's definitely supported by sdk 8, and your comment in that issue said not to worry about older unsupported sdks.

Reply:
According to Google it was added in .Net 5.